modules/video/src/lkpyramid.cpp

```diff
@@ -1607,8 +1605,8 @@ cv::Mat cv::estimateRigidTransform( InputArray src1, InputArray src2, bool fullA
     for( k = 0; k < RANSAC_MAX_ITERS; k++ )
     {
         int idx[RANSAC_SIZE0];
-        Point2f a[RANSAC_SIZE0];
-        Point2f b[RANSAC_SIZE0];
+        Point2f *a =new Point2f[RANSAC_SIZE0];
```

Comment:
`std::vector<Point2f> a(RANSAC_SIZE0);` may be better, since it
abstracts away new/delete.

Reply:
I agree; also, those vectors should better be placed outside of the loop.